src/readMSH.cpp

```diff
@@ -44,25 +37,26 @@ npe_arg(filename, std::string)
 npe_default_arg(dtypef, npe::dtype, "float")
 
 npe_begin_code()
+  Eigen::MatrixXd v;
+  Eigen::MatrixXi t;
+
+  //NOTE: readMSH only support vertices as matrix of doubles. We instead cast the output accordingly.
+  bool ret = igl::readMSH(filename, v, t);
```

Comment:
readMSH refactored in PR: https://github.com/libigl/libigl/commit/8c95d1044ed509112108d168653ce7a1839c1508#diff-14358970970f78facdd59d4179315761ff7f7a3114d962c272fe7237a987b8b2

Note that readMSH is not templated, instead only accepting MatrixXd vertex matrix as input.